CHANGES.md

```diff
@@ -1,3 +1,24 @@
+Version 6.1.0 (3rd May 2018)
+--------------------------------
+
+#### Major Changes
+
+* ReportEvent in cva package renamed to ReportEventRecord
+
+#### Minor changes
+
+* Migrations from participants 1.0.3 to 1.1.0
+* Removed participants backwards migrations
+
+Version 6.0.9 (1st May 2018)
```

Comment:
@priesgo have you released 6.0.9 to pypi ?

Reply:
I did today, not sure if after or before this message.